include/contest.h

```diff
@@ -1,6 +1,24 @@
 #ifndef GUARD_CONTEST_H
 #define GUARD_CONTEST_H
 
+#include "palette.h"
```

Comment:
Why is this include in the header?

Reply:
Because the `PLTT_BUFFER_SIZE` constant is used as part of a struct definition.